coverage_assessment.R

```diff
@@ -19,13 +19,38 @@ nut <- get_data()
 
 # Problem 1: Describe the data ----
 
-
-
-
-
-
-
-
+#How many children in the baseline 
+sum(nut$survey_round == "Baseline")
+#Number of children in endline 6552
+sum(nut$survey_round == "Endline")
+nrow (survey_round)
```

Comment:
your code is erroring at this line.

If you want to use the `nrow()` command to get the number of rows, then you need to provide the whole data.frame like this:

```R
nrow(nut)
```

If you want to get the length/number of values in the `survey_round` variable in the `nut` object, then you should do this:

```R
length(nut$survey_round)
```